data/json/npcs/robofac/robofac_intercom/robofac_intercom.json

```diff
@@ -640,7 +616,7 @@
     "responses": [
       { "text": "…", "topic": "TALK_ROBOFAC_INTERCOM_ARMOR_NEUTRAL" },
       {
-        "text": "[INT 10] The armor seemed warped.  Whatever you're using to reinforce the core of the pieces isn't strong enough.",
+        "text": "[INT 10] The armor pieces seemed warped.  Whatever you're using to reinforce the core of the material isn't strong enough.",
```

Comment:
core here refers to the center of the armor (like a tootsie pop). Since its plastic with aluminum in the center the idea is that the aluminum is too soft meaning that the pieces warp. So I don't think material is the right word.